index.html

```diff
@@ -15,90 +15,145 @@
+            </div>
+
             <div class="multi-post">
                 <img src="./images/works.png" alt="projects-image">
-                <h3>Multi-Post Stories</h3>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                    industry's standard dummytext ever since the1500s, when an unknownprinter took a standarddummy
-                    text.
-                </p>
-                <div class="keywords">
-                    <ul>
-                        <li>
-                            <a href="#">css</a>
-                        </li>
-                        <li>
-                            <a href="#">html</a>
-                        </li>
-                        <li>
-                            <a href="#">bootstrap</a>
-                        </li>
-                        <li>
-                            <a href="#">Ruby</a>
-                        </li>
-                    </ul>
+                <div class="multi-post-content">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard dummytext ever since the1500s, when an unknownprinter took a standarddummy
+                        text.
+                    </p>
+                    <div class="keywords">
+                        <ul>
+                            <li>
+                                <a href="#">css</a>
+                            </li>
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit" class="primary-btn">See Project</button>
                 </div>
-                <button type="submit">See Project</button>
+
             </div>
         </div>
         <div class="projects">
             <div class="project-background">
                 <div class="project-foreground">
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                        industry's standard</p>
+                    <div class="projects-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the
+                            industry's standard</p>
+                    </div>
+
                     <div class="project-keyword">
```

Comment:
Please kindly fit the content of the work section into the screen properly in order to match the figma as we do not want to give users at this breakpoint `1176px` a bad UX.

see your current design

[screen-recorder-sat-dec-10-2022-09-48-09.webm](https://user-images.githubusercontent.com/84930355/206841968-e8a9f7a0-81e7-4e3f-ab99-bc04889907a5.webm)


see figma design


<img width="284" alt="desktoooooop" src="https://user-images.githubusercontent.com/84930355/206842006-83dcb376-34a0-4ccb-9a18-6001ecb07ed8.png">